oauthlib/oauth2/rfc8628/endpoints/device_authorization.py

```diff
@@ -39,6 +40,7 @@ def __init__(
         expires_in=1800,
         interval=None,
         verification_uri_complete=None,
+        user_code_generator: Callable[[None], str] = None
```

Comment:
I thought it would be nice to introduce typing to this repo to make objects easier to understand without having to check during run time